clusters/resource_library.go

```diff
@@ -12,14 +12,24 @@ import (
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
 )
 
-func ResourceLibrary() common.Resource {
-	libraySdkSchema := common.StructToSchema(compute.Library{}, func(m map[string]*schema.Schema) map[string]*schema.Schema {
-		m["cluster_id"] = &schema.Schema{
-			Type:     schema.TypeString,
-			Required: true,
-		}
-		return m
+type LibraryResource struct {
+	compute.Library
+}
+
+func (LibraryResource) Aliases() map[string]map[string]string {
+	return map[string]map[string]string{}
+}
+
```

Comment:
Is this still needed? I thought we made aliases optional?

Reply:
ah, yes, good catch